### DISCUSSION TITLE
writing MF6 input files of single packages

### DISCUSSION THREAD

**Discussion by marcowhites** [modflowpy/flopy#1349] [Q&A]
Hi,
I am building as big transient model in MF6. The recharge file is about 2Gb. Since I am still calibrating the model parameters, I was wondering if there's a way to write only the input file of just a specific package instead of writing all the input files for all the packages (.write_simulation()) every single time. This could save me a lot of time since it takes a while to write the recharge file. 
I could not find an answer in the manual and in the tutorials  

Thanks and best regards,
Marco Bianchi (BGS)

**Comment by jlarsen-usgs** [comment]
@marcowhites 

You can write individual packages instead of the entire simulation: here is a quick example:

```
import flopy
import os

sim_ws = os.path.join("examples", "data", "mf6-freyberg")
sim = flopy.mf6.MFSimulation.load(sim_ws=sim_ws)
sim.set_sim_path(".")

ml = sim.get_model("freyberg")
ml.rch.write()
```
